src/ukify/ukify.py

```diff
@@ -449,7 +449,26 @@ class PeError(Exception):
 
 def pe_add_sections(uki: UKI, output: str):
     pe = pefile.PE(uki.executable, fast_load=True)
-    assert pe.FILE_HEADER.PointerToSymbolTable != 0 or len(pe.__data__) % pe.OPTIONAL_HEADER.FileAlignment == 0
+
+    # Old stubs might have been stripped, leading to unaligned raw data values, so let's fix them up here.
+    for i in range(len(pe.sections)):
+        oldp = pe.sections[i].PointerToRawData
+        oldsz = pe.sections[i].SizeOfRawData
+        pe.sections[i].PointerToRawData = round_up(pe.sections[i].PointerToRawData, pe.OPTIONAL_HEADER.FileAlignment)
+        pe.sections[i].SizeOfRawData = round_up(pe.sections[i].SizeOfRawData, pe.OPTIONAL_HEADER.FileAlignment)
+        padp = pe.sections[i].PointerToRawData - oldp
+        padsz = pe.sections[i].SizeOfRawData - oldsz
+
+        for j in range(i + 1, len(pe.sections)):
+            pe.sections[j].PointerToRawData += padp + padsz
+
+        pe.__data__ = pe.__data__[:oldp] + padp * b'\0' + pe.__data__[oldp:oldp+oldsz] + padsz * b'\0' + pe.__data__[oldp+oldsz:]
```

Comment:
Instead of going into the the privates of `pe`, could this be done with `pe.get_data` and `pe.set_data_bytes`? They look like pretty thin wrappers around `__data__`, but I haven't tested this.

Reply:
Do not use `pe.get_data()`, it just returns the structure, which in case of `pe` ends up being the headers.